code/modules/mob/living/carbon/life.dm

```diff
@@ -8,28 +8,33 @@
 		damageoverlaytemp = 0
 		update_damage_hud()
 
-	if(stat != DEAD) //Reagent processing needs to come before breathing, to prevent edge cases.
-		handle_organs()
+	if(!IsInStasis())
 
-	. = ..()
+		if(stat != DEAD) //Reagent processing needs to come before breathing, to prevent edge cases.
+			handle_organs()
 
-	if (QDELETED(src))
-		return
+		. = ..()
 
-	if(.) //not dead
-		handle_blood()
+		if (QDELETED(src))
+			return
 
-	if(stat != DEAD)
-		var/bprv = handle_bodyparts()
-		if(bprv & BODYPART_LIFE_UPDATE_HEALTH)
-			updatehealth()
-			update_stamina()
+		if(.) //not dead
+			handle_blood()

```

Comment:
This can kill you if you run out of blood or have low blood.
https://github.com/tgstation/tgstation/blob/5b89dea70f84fe72a653c4d65fbf056206764fc3/code/modules/mob/living/blood.dm#L66-L67
https://github.com/tgstation/tgstation/blob/5b89dea70f84fe72a653c4d65fbf056206764fc3/code/modules/mob/living/blood.dm#L71-L73